R/plotting.R

```diff
@@ -1223,13 +1223,13 @@ lineage_sunburst <- function(prot, lineage_column = "Lineage",
         group_by_at(levels_vec) %>%
         summarise(size = n())
     protLevels <- protLevels %>% arrange()
-    tree <- d3_nest(protLevels, value_cols = "size")
+    tree <- .data$d3_nest(protLevels, value_cols = "size")
```

Comment:
This is actually looking for an additional import. Adding an Rd tag declaring this import to fix:
```
@importFrom d3r d3_nest
```